src/server.rs

```diff
@@ -159,6 +159,10 @@ pub async fn run(
             &format!("{}/detection/content", TEXT_API_PREFIX),
             post(detection_content),
         )
+        .route(
+            &format!("{}/detection/context-docs", TEXT_API_PREFIX),
```

Comment:
we are renaming this?

Reply:
Yes, we agreed on renaming to `/detection/context`. I've also updated any mentions of it in the documentation.